agent/acl_oss.go

```diff
@@ -7,8 +7,13 @@ import (
 	"github.com/hashicorp/serf/serf"
 
 	"github.com/hashicorp/consul/acl"
+	"github.com/hashicorp/consul/api"
 )
 
 func serfMemberFillAuthzContext(m *serf.Member, ctx *acl.AuthorizerContext) {
 	// no-op
 }
+
+func agentServiceFillAuthzContext(s *api.AgentService, ctx *acl.AuthorizerContext) {
+	// no-op
+}
```

Comment:
I thought we generally used a method for this, but I see there is prior art above here. Any idea why we use a function sometimes and a method the rest of the time?

Reply:
Here it's because the type is from the `api` package (above is from `serf`) so it wouldn't be appropriate to put it there.